{
    "config": {
        "abort": {
            "already_configured": "La cuenta ya est\u00e1 configurada",
            "reauth_failed_existing": "No se pudo actualizar la entrada de configuraci\u00f3n, por favor, elimina la integraci\u00f3n y vuelve a configurarla.",
            "reauth_successful": "La autenticaci\u00f3n se volvi\u00f3 a realizar correctamente",
            "unknown": "Error inesperado"
        },
        "error": {
            "cannot_connect": "No se pudo conectar",
            "invalid_api_key": "Clave API no v\u00e1lida",
            "not_main_key": "Se detect\u00f3 un tipo de clave API incorrecto, utiliza la clave API 'principal'",
            "reauth_failed_matching_account": "La clave de API que proporcionaste no coincide con el ID de cuenta para la configuraci\u00f3n existente.",
            "unknown": "Error inesperado"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "api_key": "Clave API"
                },
                "description": "Debes proporcionar una nueva clave API 'principal' de UptimeRobot",
                "title": "Autenticaci\u00f3n caducada para {name}"
            },
            "user": {
                "data": {
                    "api_key": "Clave API"
                },
                "description": "Debes proporcionar la clave API 'principal' de UptimeRobot"
            }
        }
    },
    "entity": {
        "sensor": {
            "monitor_status": {
                "state": {
                    "down": "Ca\u00eddo",
                    "not_checked_yet": "A\u00fan no se ha comprobado",
                    "pause": "Pausar",
                    "seems_down": "Parece ca\u00eddo",
                    "up": "Arriba"
                }
            }
        }
    }
}